neptunecontrib/api/explainers.py

```diff
@@ -0,0 +1,203 @@
+import neptune
+
+from neptunecontrib.api import log_chart
+
+__all__ = [
+    'log_explainer',
+    'log_local_explanations',
+    'log_global_explanations'
+]
+
+
+def log_explainer(filename, explainer, experiment=None):
+    """Logs dalex explainer to Neptune.
+
+    Dalex explainer is pickled and logged to Neptune.
+
+    Args:
+        filename (:obj:`str`): filename that will be used as an artifact's destination.
+        explainer (:obj:`dalex.Explainer`): and instance of dalex explainer
+        experiment (:obj:`neptune.experiments.Experiment`, optional, default is ``None``):
+            | For advanced users only. Pass Neptune
+              `Experiment <https://docs.neptune.ai/neptune-client/docs/experiment.html#neptune.experiments.Experiment>`_
+              object if you want to control to which experiment data is logged.
+            | If ``None``, log to currently active, and most recent experiment.
+
+    Examples:
+        Start an experiment::
+
+            import neptune
+
+            neptune.init(api_token='ANONYMOUS',
+                         project_qualified_name='shared/dalex-integration')
+            neptune.create_experiment(name='logging explanations')
+
+        Train your model and create dalex explainer::
+
+            ...
+            clf.fit(X, y)
+
+            exp = dx.Explainer(clf, X, y, label="Titanic MLP Pipeline")
```

Comment:
Can we maybe call it `expl`, so that it doesn't get mixed up with our experiments?

Reply:
Here and in other examples.

Reply:
done